### DISCUSSION TITLE
Can't view repo in github

### DISCUSSION THREAD

**Discussion by Xapp00** [Part-II-Computational-Physics/.github#76] [Q&A]
Hi, 
I'm currently having issues viewing my work in github. I've done everything in VS code but  after uploading the repo wouldn't show up in git, but ok if opened with VS code. Do you know how to fix this?

**Comment by AustenLamacraft** [comment]
GH is probably struggling with the very large outputs you have. If you output the average accuracy per [this comment](https://github.com/orgs/Part-II-Computational-Physics/discussions/75#discussioncomment-5295949) that may help. Plus there's a plot with like 1000 labels in the legend
